src/providers/address/address.ts

```diff
@@ -40,6 +40,7 @@ export class AddressProvider {
     str: string,
     network: string = 'livenet'
   ): CoinNetwork {
+    if (!str) return null;
```

Comment:
That function should never been called without 'str'. We should check the source of the problem.

Reply:
I think it is not necessary anymore since we have a conditional here: https://github.com/bitpay/wallet/pull/11687/files#diff-e20f5602886c255ee30a38778ad8d5c2a889c2dfe2fedf1126d732efd59e9f82R86